types/socket/events.ts

```diff
@@ -35,11 +34,15 @@ interface PlayerToServerEvents {
     spectatorID: string
   ) => void;
 
+  QUIT_TOURNAMENT: (
```

Comment:
What is the purpose for this event exactly? To allow the host to quit and destory an tournament?

If so then you will atleast need some more data to be sent to make it slightly more secure (primarily the hostID) otherwise any user could send the tournament code to destory a tournment.

Reply:
Also this event would technically fit under the HostToServerEvents category (may not exist atm).

Reply:
> Also this event would technically fit under the HostToServerEvents category (may not exist atm).

I'll add this in a later PR